lib/sprig/seed/attribute.rb

```diff
@@ -54,7 +54,7 @@ def computed_value?(value)
       end
 
       def computed_value_regex
-        /<%[=]?(.*)%>/
+        /(<%[=]?(.*?)%>)/
```

Comment:
@efatsi yep it can be just `=?`. Good spot. Changing it now. Requiring the equals is more subjective. I'm less opposed to changing that now, however that would require a major release since some apps might be using it without. Should have just been less flexible from the get go. I'll sit on this for a bit.